### DISCUSSION TITLE
JDownloader-2 not found after installation

### DISCUSSION THREAD

**Discussion by NougatPistache** [elementary/triage#560] [Q&A] [answered]
Hello

On Elementary OS, to install JDownloader-2, I typed the following commands:

```
sudo apt update
sudo apt install snapd
```

`sudo snap install jdownloader2`

I got a message telling me that JDownloader-2 has been successfully installed.
I'm looking in my program library, but JD is nowhere to be found.

Did I make a mistake somewhere?

Thank you

**Comment by jlnr** [comment, marked as answer]
Whenever I first installed either snapd or flatpak on a distro that didn't ship with them, I had to log out and back in to have the "new kind of apps" show up in the menu. Have you tried that and did it help?

**Comment by NougatPistache** [comment]
The 4th disconnection was the right one. 
Thank you
